app/models/reservation.rb

```diff
@@ -103,6 +103,25 @@ class Reservation < ActiveRecord::Base
       .select('reservations.*, equipment_models.category_id as category_id')
   }
 
+  # for status modifying jobs
+  def self.deletable_missed
```

Comment:
I don't think there's any reason why you can't just assign a scope chain to a Query Object, although I guess ideally you'd want the Query Object to be independent of the Model.

Re: responsibility, I feel like the responsibility of knowing the details of what to handle does belong in the job, because that's what's doing it. In other words, you can think of `DeleteMissedJob` as a `MissedReservationDeleter`, and, since we have a very specific query for missed reservations that can be deleted (and this is the only place where it's used), I'm inclined to leave the details of that query in the Deleter vs the overall Model.

Similar to my previous comment, feel free to disagree, I'm just trying to figure out how best to think about all this stuff.